VRDR/DeathRecord.cs

```diff
@@ -282,6 +301,19 @@ public DeathRecord()
             string[] decedent_profile = { "http://hl7.org/fhir/us/vrdr/StructureDefinition/VRDR-Decedent" };
             Decedent.Meta.Profile = decedent_profile;
 
+            // Start with an empty race and ethinicity observation
```

Comment:
Why doesn't this use the `CreateRaceEthnicityObs` method? Looks like it duplicates the code.

Reply:
The CreateRaceEthnicityObs calls "AddReferenceToComposition" which throws a null reference exception so the function doesn't work here